tests/end2end/features/fileselect.feature

```diff
@@ -0,0 +1,35 @@
+# vim: ft=cucumber fileencoding=utf-8 sts=4 sw=4 et:
+
+Feature: Selecting files
+
+    Background:
+        Given I have a fresh instance
```

Comment:
This slows down things a lot because it starts a new instance for every test. I just noticed this is in `editor.feature` as well though (via 6519f500a9f779ce868d1dd9556902e607cde6ab)...

It's not actually needed, and removing it as much as possible (62cf48fc779a22617957bd18272a55a04a8ffc2d) now makes the editor tests run in 8s instead of 30s on my machine...

Probably irrelevant for you with the comment above, though.

Reply:
Yep, since merged with `editor.feature`